### PR TITLE
[docker] Update docker scripts and docs

### PR DESCRIPTION
**Pull request by zach2good**
Docker was broken, this fixes it

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: tools/wait_for_db_then_launch.sh [hunk 1]
```diff
@@ -13,11 +14,12 @@ else
     XI_DB_VER=`git rev-parse --short=4 HEAD`
 fi
 echo -e "\n#DB_VER: ${XI_DB_VER}" >> ../conf/version.conf
+
 # Update database if needed
-python3 dbtool.py update
+python3 tools/dbtool.py update
```
Comment (zach2good) [inline tools/wait_for_db_then_launch.sh:19]: This is one of the things that was broken

File: Dockerfile [hunk 2]
```diff
@@ -8,20 +8,25 @@ ENV DEBIAN_FRONTEND=noninteractive
...
 
 # Update and install all requirements as well as some useful tools such as net-tools and nano
-RUN apt update && apt install -y net-tools nano software-properties-common git python3 python3-pip g++-9 cmake make libluajit-5.1-dev libzmq3-dev libssl-dev zlib1g-dev mariadb-server libmariadb-dev luarocks
+RUN apt update && apt install -y net-tools nano software-properties-common git python3 python3-pip clang-11 cmake make libluajit-5.1-dev libzmq3-dev libssl-dev zlib1g-dev mariadb-server libmariadb-dev luarocks
 
-# Copy everything from the host machine topaz folder to /topaz
-ADD . /topaz
+# Use Clang 11
+ENV CC=/usr/bin/clang-11
+ENV CXX=/usr/bin/clang++-11
```
Comment (zach2good) [inline Dockerfile:23]: And this

**Review by zach2good — COMMENTED** [review]
Highlighting the bits that were broken

**Comment by Xaver-DaRed** [comment]
Such a weird CI error... Anyway, LGTM. Also like the removal of branding.

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**